style.css

```diff
@@ -23,23 +26,37 @@
 }
 
 .container {
-  background-image: url(/img/background.png);
+  background-image: url(/img/background2.png);
+  width: auto;
+  height: auto;
+  margin: none;
+  margin-top: 40px;
+}
```

Comment:
- [ ] Well done on implementing previous reviewer suggestions. 👍 But, there is still some issue. The background image is repeated twice in the same frame, so there are some there `extra shapes` coming that are not in the design guideline.
  - I suggest you use  ` background-repeat: no-repeat;` and `background-size: cover;` in the `container` class to fix this issue. Kindly consider this.
  - ***Your Design***
  ![image](https://user-images.githubusercontent.com/60083437/219509753-8ab58ec3-56df-40aa-8e7a-f7357e5fc9ba.png)